packages/buttons/src/elements/Button.tsx

```diff
@@ -36,14 +38,17 @@ const ButtonComponent = forwardRef<HTMLButtonElement, IButtonProps>((props, ref)
     }
 
     computedProps = buttonGroupContext.getButtonProps({
-      item: props.value,
-      focusRef: React.createRef(),
       isSelected: props.value === buttonGroupContext.selectedItem,
       ...computedProps
     });
+    computedRef = mergeRefs([
+      // a `ref` is added by the `useSelection` container which we need to merge with the forwarded `ref`
+      (computedProps as IButtonProps & { ref: MutableRefObject<HTMLButtonElement> }).ref,
+      ref
+    ]);
   }
 
-  return <StyledButton ref={ref} {...computedProps} />;
+  return <StyledButton ref={computedRef} {...computedProps} />;
```

Comment:
If I'm reading this right, `computedRef` will be stomped over by `computedProps.ref` since `ref` isn't destructured from `computedProps`. easy fix tho. :) 

```suggestion
  return <StyledButton {...computedProps} ref={computedRef} />;
```

Reply:
Good catch!

Reply:
Updated in [cc74e35](https://github.com/zendeskgarden/react-components/pull/1576/commits/cc74e35ba3295adda89a6f79203c2c6d602c8614).